styles/_base.scss

```diff
@@ -119,7 +119,6 @@ hr {
     left: 0;
     right: 0;
     bottom: 0;
-    background: rgba(0, 0, 0, 0.3);
```

Comment:
This means that the behavior would still remain the same, where clicking outside the timer area will close the selector. Wouldn't this look and feel a bit strange? When the rest of the page looks normal but is unclickable?

Perhaps we should remove the behavior altogether and let the use exit the selector by pressing escape, enter or one of the preset options?